src/displayapp/screens/settings/SettingSetDate.cpp

```diff
@@ -80,9 +81,9 @@ SettingSetDate::SettingSetDate(Pinetime::Applications::DisplayApp* app,
 
   btnSetTime = lv_btn_create(lv_scr_act(), nullptr);
   btnSetTime->user_data = this;
-  lv_obj_set_size(btnSetTime, 120, 48);
+  lv_obj_set_size(btnSetTime, LV_HOR_RES - 35, 48);
```

Comment:
LV_HOR_RES_MAX is more efficient, because it is can be compile time evaluated.

```suggestion
  lv_obj_set_size(btnSetTime, LV_HOR_RES_MAX - 35, 48);
```